Sources/SkyFloatingLabelTextFieldWithIcon.swift

```diff
@@ -12,9 +12,50 @@
 import UIKit
 
 /**
+ Identify the type of icon. 
+    - font: Set your icon by setting the font of iconLabel
+    - image: Set your icon by setting the image of iconImageView
+ */
+public enum IconType: Int {
+    case font
+    case image
+}
+
+/**
  A beautiful and flexible textfield implementation with support for icon, title label, error message and placeholder.
  */
 open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
+    
+    @IBInspectable
+    var iconTypeAdapter: Int {
```

Comment:
We should log a warning or error somehow if the user tries to an image when the type is `.font` or iconText when the type is `.image`